apps/meteor/client/lib/starMessage.ts

```diff
@@ -0,0 +1,50 @@
+import type { IMessage } from '@rocket.chat/core-typings';
+import { Meteor } from 'meteor/meteor';
+
+import { ChatMessage, ChatSubscription } from '../../app/models/client';
+import { settings } from '../../app/settings/client';
+import { t } from '../../app/utils/lib/i18n';
+import { dispatchToastMessage } from './toast';
+
+export const starMessage = (message: IMessage, starred: boolean) => {
+	const uid = Meteor.userId();
```

Comment:
if this is a new function, shouldn't we be able to just pass the uid instead of depending on meteor?

Reply:
As we intent to drop minimongo collections, the whole content of this function is going to be outdated soon. Also, there is a change coming from #34017 that might stale it.